### PR TITLE
Support decoding JSONs in embedded panels in dashboards

### PR DESCRIPTION
**Pull request by jsoriano**
Add support for decoding JSONs in embedded panels in dashboards, also add support for deconding control group inputs.

### CODE REVIEW

**Comment by jsoriano** [comment]
test integrations

File: internal/builder/dashboards.go [hunk 1]
```diff
@@ -80,11 +98,54 @@ func encodedSavedObject(data []byte) ([]byte, bool, error) {
...
+		if !ok {
+			continue
+		}
+		embeddableConfig, ok := embeddableConfigValue.(map[string]any)
+		if !ok {
+			return nil, false, fmt.Errorf("embeddable config is not a map, found %T", embeddableConfigValue)
+		}
+		embeddableConfig, embeddableChanged, err := encodeObjectMapStr(common.MapStr(embeddableConfig))
+		if err != nil {
+			return nil, false, fmt.Errorf("econdign embedded object failed: %w", err)
```
Comment (mrodm) [inline internal/builder/dashboards.go:139]: Nit.
```suggestion
			return nil, false, fmt.Errorf("encoding embedded object failed: %w", err)
```

**Review by mrodm — COMMENTED** [review]
Looking into the created PR in integrations, there are some errors raised. For instance in `beaconing`:

> Error: checking package failed: building package failed: encoding dashboards failed: encoding /opt/buildkite-agent/builds/bk-agent-prod-gcp-1715187930001048022/elastic/integrations/build/packages/beaconing/1.2.0/kibana/dashboard/beaconing-5c610e50-2d67-11ee-b990-0d17f5be0d77.json: expected list of panels, found string
[beaconing] run_tests_package failed

File: internal/builder/dashboards.go (hunk 1 above)
Comment (jsoriano) [reply internal/builder/dashboards.go]: Thanks, fixed.

**Comment by jsoriano** [comment]
test integrations

**Pull request merged**